src/FsAutoComplete.Core/CommandResponse.fs

```diff
@@ -342,6 +376,36 @@ module CommandResponse =
     | GenericError errorMessage -> error serialize errorMessage //compatibility with old api
     | ProjectNotRestored project -> errorG serialize (ErrorData.ProjectNotRestored { Project = project }) "Project not restored" 
 
+  let workspacePeek (serialize : Serializer) (found: FsAutoComplete.WorkspacePeek.Interesting list) =
+    let mapInt i =
+        match i with
+        | FsAutoComplete.WorkspacePeek.Interesting.Directory (p, fsprojs) ->
+            WorkspacePeekFound.Directory { WorkspacePeekFoundDirectory.Directory = p; Fsprojs = fsprojs }
+        | FsAutoComplete.WorkspacePeek.Interesting.Solution (p, sd) ->
+            let rec item (x: FsAutoComplete.WorkspacePeek.SolutionItem) =
+                let kind =
+                    match x.Kind with
+                    | FsAutoComplete.WorkspacePeek.SolutionItemKind.Unknown
+                    | FsAutoComplete.WorkspacePeek.SolutionItemKind.Unsupported ->
+                        None
+                    | FsAutoComplete.WorkspacePeek.SolutionItemKind.MsbuildFormat msbuildProj ->
+                        Some (WorkspacePeekFoundSolutionItemKind.MsbuildFormat { 
+                            WorkspacePeekFoundSolutionItemKindMsbuildFormat.Configurations = [] 
+                        })
+                    | FsAutoComplete.WorkspacePeek.SolutionItemKind.Folder(children, files) ->
+                        let c = children |> List.choose item
+                        Some (WorkspacePeekFoundSolutionItemKind.Folder { 
+                            WorkspacePeekFoundSolutionItemKindFolder.Items = c
+                            Files = files
+                        })
+                kind
+                |> Option.map (fun k -> { WorkspacePeekFoundSolutionItem.Guid = x.Guid; Name = x.Name; Kind = k })
+            let items = sd.Items |> List.choose item
+            WorkspacePeekFound.Solution { WorkspacePeekFoundSolution.Path = p; Items = items; Configurations = [] }
+
+    let data = { WorkspacePeekResponse.Found = found |> List.map mapInt }
+    serialize { Kind = "project"; Data = data }
```

Comment:
We try to use different Kind for each response, `project` is used by project command